<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreeningTab</class>
 <widget class="QWidget" name="ScreeningTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>740</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Screening Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Movie:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="movieCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hall:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="hallCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Time:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="startTimeInput"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Time:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="endTimeInput"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="screeningTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Screening ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Movie</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hall</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Time</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addScreeningBtn">
       <property name="text">
        <string>Add Screening</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateScreeningBtn">
       <property name="text">
        <string>Update Screening</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteScreeningBtn">
       <property name="text">
        <string>Delete Screening</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshScreeningBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
